internal/cli/doctor.go

```
@@ -157,17 +202,70 @@ func printCapabilityMatrix(w io.Writer, targets []render.Renderer, caps []render
+		}
+		return
+	}
+
+	fmt.Fprintln(w, "≈ = same underlying feature, expressed via a different harness-native mechanism (not a gap):")
+	for _, s := range substituted {
+		fmt.Fprintf(w, "%s  %s — via %s\n", s.target, s.cap, s.via)
+	}
+}
+
+func printMarkdownCapabilityTable(w io.Writer, targets []render.Renderer, caps []render.Capability, mark func(int, render.Capability) string) {
+	fmt.Fprint(w, "| capability |")
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprint` is not checked

(errcheck)